etcdserver/server.go

```diff
@@ -1435,20 +1440,20 @@ func (s *EtcdServer) TransferLeadership() error {
 		return nil
 	}
 
-	if !s.isMultiNode() {
+	if s.cluster == nil || len(s.cluster.VotingMemberIDs()) <= 1 {
```

Comment:
probably still encapsulate this logic into a func.

Reply:
ok.